app/electron/main.js

```diff
@@ -114,6 +150,7 @@ async function createWindow() {
     // before the DOM is ready
     win.webContents.once("dom-ready", () => {
       // win.webContents.openDevTools(); // see https://github.com/reZach/secure-electron-template/issues/48
+      // eslint-disable-next-line global-require
```

Comment:
Do not disable eslint, let the warning be there, as a reminder that we have used global requires, we could refactor this to not require global require, but that might cause unnecessary overhead when not used